### ISSUE TITLE
at least one output is not set while no more ensemble steps can be made

### ISSUE THREAD

**Issue by thunder95** [triton-inference-server/server#6505]
My configuration looks like the following:

`name: "ner"
platform: "ensemble"
max_batch_size : 8

#input to the model 
input [
    {
        name: "text"
        data_type: TYPE_STRING
        dims: [ -1 ] 
        # -1 means dynamic axis, aka this dimension may change 
    }
]

#output of the model
output [
    {
        name: "outputs"
        data_type: TYPE_STRING
        dims: [ -1 ]
    }
]

#Type of scheduler to be used
ensemble_scheduling {
    step [
        {
            model_name: "tokenizer"
            model_version: -1
        input_map {
            key: "text"
            value: "text"
        }
        output_map [
        {
            key: "input_ids"
            value: "INPUT_IDS"
        },
	{
            key: "attention_mask"
            value: "ATTENTION_MASK"
        },
        {
            key: "token_type_ids"
            value: "TOKEN_TYPE_IDS"
        },
        {
	    key: "special_tokens_mask"
            value: "SPECIALTOKENS_MASK"
	}
        ]
        },
        {
            model_name: "model"
            model_version: -1
        input_map [
		 {
            key: "input_ids"
            value: "INPUT_IDS"
        },
        {
            key: "attention_mask"
            value: "ATTENTION_MASK"
        },
        {
            key: "token_type_ids"
            value: "TOKEN_TYPE_IDS"
        }
        ]
        output_map [
            {
                key: "logits"
                value: "Logits"
            }
        ]
        },
        {
            model_name: "postprocess"
            model_version: -1
        input_map [
        {
            key: "logits"
            value: "Logits"
        },
        {
            key: "special_tokens_mask"
            value: "SPECIALTOKENS_MASK"
        }
        ]
        output_map [
            {
                key: "outputs"
                value: "outputs"
            }
        ]
        }
    ]
}
version_policy{all {} }
`

The problem is supposed to be "special_tokens_mask", which was created at the first step and used at the last step. Triton server can run successfully, but it threw this error when I send request .

The idea of this pipeline just followed the ner pipeline of transformers, where "special_tokens_mask" is wanted at the postprocessing step. 

If I used other variables like "input_ids", which is transferred along this pipeline without break, this error would not be appear.

Please help me.
